website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,192 @@
+## Usage
+
+### When to use
+
+- When the list of options is large and search is required to help filter the list.
+- When selecting more than one option is necessary.
```

Comment:
The native HTML select supports multiple selection so we may want to rethink this one a bit.

You could argue that the SuperSelect provides a better or more consistent user interface and/or visual style than HDS Select since it contains an HTML select under the hood...

See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#multiple

Thoughts? @heatherlarsen @andgen404

Reply:
I think we can remove this.

Reply:
@majedelass I can delete for you if that's ok.